conda/recipes/rmm/meta.yaml

```diff
@@ -25,6 +25,7 @@ requirements:
     - librmm {{ version }}
     - setuptools
     - cython >=0.29,<0.30
+    - spdlog >=1.4.2
```

Comment:
Does this also need to go in the [`conda/rmm_dev_cuda*.yml` files](https://github.com/rapidsai/rmm/tree/branch-0.15/conda/environments)?